### PR TITLE
Hdxdsys 731 update RTD data usage guide

### PR DESCRIPTION
**Pull request by turnerm**
This PR mostly covers the text for the data usage portion of the docs. Some other changes:
- Categories have been broken up to separate files to make them a bit more manageable
- Tables are all in yaml files to avoid formatting issues
-  Small changes to index.md to align names and ordering with the data grid
- Briar and Mike will help me resolve the few remaining to-dos, so those will be gone shortly 

I still need to do the parameter tables, that will be for another PR

### CODE REVIEW

**Comment by turnerm** [comment]
@danmihaila could Briar (@b-j-mills) be added to this repo? I'd like to make them a reviewer

**Comment by alexandru-m-g** [comment]
@turnerm @b-j-mills @danmihaila , I've added Briar to the repo

File: docs/data_usage_guides/enum_parameters/population_group_parameters.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- population group: RST
+  description: 'resettled refugees'
+- population group: NAT
+  description: 'naturalized refugees'
+- population group: RDP
+  description: 'returned IDPs'
+- population group: POC
+  description: 'persons of concern'
+- population group: RRI
+  description: 'TODO: what is this'
```
Comment (mcarans) [inline docs/data_usage_guides/enum_parameters/population_group_parameters.yaml:30]: I remember now why I made this - UNHCR has Returned Refugees (RET) and Returned IDPS (RDP). HNO data only has returnees (the combination of the two) so I used RRI for Returned Refugees and IDPs.

File: docs/data_usage_guides/metadata.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+The names and p-codes are read in from a [global p-code list](https://data.humdata.org/dataset/global-pcodes)
+taken from the common operational dataset (COD) gazetteers administrative
+boundaries.
+The p-code field is unique only in combination with reference_period_start,
+since p-codes may be reused in different versions of geographical metadata.
+
+When bringing in data from sub-categories, HDX HAPI uses p-code matching
+from [`hdx-python-country`](https://hdx-python-country.readthedocs.io/en/latest/).
+Thie can involve .... TODO
```
Comment (mcarans) [inline docs/data_usage_guides/metadata.md]: HDX HAPI utilises p-code handling provided by the [`hdx-python-country`](https://hdx-python-country.readthedocs.io/en/latest/) library. When bringing in data from sub-categories, p-codes are used if present. Sometimes, these p-codes may not strictly follow the standard with variations such as shorter or longer country or admin unit codes. The hdx-python-country library has a p-code length matching algorithm to ensure correct admin units are determined. Where p-codes are unavailable in the data, it has the capability to match admin unit names to p-codes aided by both automatic phonetic matching and manual overrides.

File: docs/data_usage_guides/metadata.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+### WFP Market <a id="wfp-market"></a>
+
+**Used in:**
+[`Food Prices`](food_security_and_nutrition.md#food-prices)
+
+Markets are defined as the physical locations where buyers and sellers
+come together to trade goods and services. This table is populated from the
+food prices data, which contains location names down to admin 2 as well as
+a latitude and longitude, however is not p-coded.
+We therefore p-code .... TODO
```
Comment (mcarans) [inline docs/data_usage_guides/metadata.md]: This table is populated from the food prices data, which contains location names down to admin 2 as well as
a latitude and longitude, but is not p-coded. Consequently, admin 1 and 2 names must be matched to p-codes using the algorithm provided by the hdx-python-country library which includes phonetic name matching and manual overrides.

File: docs/data_usage_guides/enum_parameters/price_flag_parameters.yaml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+- price flag: actual
+- price flag: aggregate
+- price flag: actual,aggregate
```
Comment (mcarans) [inline docs/data_usage_guides/enum_parameters/price_flag_parameters.yaml:7]: actual data is collected originally with monthly frequency; aggregated data is collected with higher frequency (daily or weekly) averaged through a weighted mean to monthly; imputed data is an estimation of missing values in the time series, obtained through the R Amelia-II package; forecasted data is a six months prediction obtained through different methods, which are reported in the API.

(We only collect actual and aggregate)

File: docs/data_usage_guides/enum_parameters/price_type_parameters.yaml [hunk 5]
```diff
@@ -0,0 +1,3 @@
+- price type: Farm Gate
+- price type: Retail
+- price type: Wholesale
```
Comment (mcarans) [inline docs/data_usage_guides/enum_parameters/price_type_parameters.yaml:3]: I have not been able to find WFP definitions for these

Comment (mcarans) [reply docs/data_usage_guides/enum_parameters/price_type_parameters.yaml:3]: FAO have this: https://www.fao.org/economic/the-statistics-division-ess/methodology/methodology-systems/price-statistics-and-index-numbers-of-agricultural-production-and-prices/4-concepts-on-price-data/en/

**Review by mcarans — APPROVED** [review]
Looks good to me

File: docs/data_usage_guides/affected_people.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+crises. It includes an estimate of the number of people by sector who require
+assistance, often referred to as People in Need (PIN), which is derived using
+the [Joint Intersectoral Analysis Framework (JIAF)](https://www.jiaf.info/).
+
+While the HNO data is
+[directly available on HDX](https://data.humdata.org/dataset/?dataseries_name=Humanitarian+Needs+Overview),
+it comes from different OCHA offices and is currently not standardised. Thus,
+HDX HAPI obtains the PIN numbers from the
+[HPC Tools API](https://api.hpc.tools/docs/v1/).
+Datasets from the HPC API will soon be available on HDX.
```
Comment (mcarans) [inline docs/data_usage_guides/affected_people.md]: They are available now eg. https://data.humdata.org/dataset/hno-data-for-nga

File: docs/data_usage_guides/coordination_and_context.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+### Transformations applied
+
+* For consistency and interoperability, we aggregate to an
+  [operational presence](https://humanitarian.atlassian.net/wiki/spaces/imtoolbox/pages/214499412/Who+does+What+Where+3W)
+  level (3W:OP, per org, sector, and admin2), even if the original 3W data is
+  more detailed (e.g. the source lists individual activities)
+* Countries that are not p-coded are aggregated to Admin 0
+* Organisation deduplication is a long-running challenge with this data, since
+  there are no unique identifiers, and organisation names may be spelled
+  different ways from different OCHA offices, or sometimes even within the same
```
Comment (mcarans) [inline docs/data_usage_guides/coordination_and_context.md]: in different ways by

File: docs/data_usage_guides/coordination_and_context.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+* Available at either Admin 0, 1, or 2 depending on the country. Please check
+  data coverage for further information
+* We cannot guarantee org consistency over time; for example, if an IMO
+  changes, the 3W might change the spelling or the whole name of an org between
+  releases
+
+## Funding <a id=”funding”></a>
+
+FTS publishes data on humanitarian funding flows as reported by donors and
+recipient organisations. It presents all humanitarian funding to a country and
```
Comment (mcarans) [inline docs/data_usage_guides/coordination_and_context.md]: and -> as well as

File: docs/data_usage_guides/coordination_and_context.md [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+
+{{ read_yaml('data_usage_guides/subcategory_details/funding_details.yaml') }}
+
+### Transformations applied
+
+* The present version of the API currently captures only funding associated
+  with an appeal. Funding data without associated appeals will be added in a
+  future version.
+* The reference period is taken from the start and end date provided in each
+  row front he original data
```
Comment (mcarans) [inline docs/data_usage_guides/coordination_and_context.md]: from the

File: docs/data_usage_guides/metadata.md [hunk 10]
```diff
@@ -0,0 +1,166 @@
...
+All of the data in HDX HAPI comes from publicly-available datasets on HDX, and
+the HDX metadata can be used refer back to the original data to see what HAPI
+has simplified or omitted.
+The two primary HDX metadata tables are
+[`dataset`](dataset) and [`resource`](resource), and they
+contain a small subset of the metadata available through the CKAN API or
+HDX library.
+
+The HAPI data records contain enough information to access the full records
+from CKAN, see the parameters tables for more details.
```
Comment (mcarans) [inline docs/data_usage_guides/metadata.md]: Use brackets ie. (see the ... details)

File: docs/data_usage_guides/enum_parameters/population_status_parameters.yaml [hunk 11]
```diff
@@ -0,0 +1,12 @@
+- population status: POP
+  description: 'population'
+- population status: AFF
+  description: 'affected'
+- population status: INN
+  description: 'inneed'
```
Comment (IanHopkinson) [inline docs/data_usage_guides/enum_parameters/population_status_parameters.yaml]: I'd say "inneed" should be "in need"

File: docs/data_usage_guides/enum_parameters/price_flag_parameters.yaml (hunk 4 above)
Comment (IanHopkinson) [reply docs/data_usage_guides/enum_parameters/price_flag_parameters.yaml:7]: How can a price be both actual and aggregate?

File: docs/data_usage_guides/enums.md [hunk 12]
```diff
@@ -0,0 +1,114 @@
...
+for their methodology and more detailed descriptions of the sub-event types.
+
+{{ read_yaml('data_usage_guides/enum_parameters/event_type_parameters.yaml') }}
+
+## Gender <a id="gender"></a>
+
+**Used in:**
+[`Baseline Population`](population_and_socio-economy.md#population),
+[`Humanitarian Needs`](affected_people.md#humanitarian-needs),
+[`Refugees & Persons of Concern`](refugees)
```
Comment (IanHopkinson) [inline docs/data_usage_guides/enums.md]: The refugees link looks different from the other two - is that correct?

File: docs/data_usage_guides/metadata.md (hunk 2 above)
Comment (IanHopkinson) [reply docs/data_usage_guides/metadata.md]: Where are p-codes introduced/described?

I've been here 6 months and I still don't really know what they are, and it is becoming too late to ask!

Comment (turnerm) [reply docs/data_usage_guides/metadata.md]: amazing thank you

**Review by IanHopkinson — APPROVED** [review]
Looks good to me and useful background

File: docs/data_usage_guides/enum_parameters/price_type_parameters.yaml (hunk 5 above)
Comment (turnerm) [reply docs/data_usage_guides/enum_parameters/price_type_parameters.yaml:3]: Yes I had found that, I was hesitant to use an FAO source to dexcribe WFP's data but there doesn't seem like a good publicly facing alternative.

File: docs/data_usage_guides/enums.md (hunk 12 above)
Comment (turnerm) [reply docs/data_usage_guides/enums.md]: oops thanks good catch

**Review by alexandru-m-g — APPROVED** [review]

File: docs/data_usage_guides/metadata.md (hunk 2 above)
Comment (b-j-mills) [reply docs/data_usage_guides/metadata.md]: @IanHopkinson there's a storymap (https://storymaps.arcgis.com/stories/dcf6135fc0e943a9b77823bb069e2578) that describes the CODs and p-codes!

**Review by b-j-mills — APPROVED** [review]
Looks good!

File: docs/data_usage_guides/enum_parameters/population_group_parameters.yaml (hunk 1 above)
Comment (b-j-mills) [reply docs/data_usage_guides/enum_parameters/population_group_parameters.yaml:30]: @turnerm should we change this TODO before merging?

**Pull request merged**